.github/workflows/daily-package-validation.yaml

```
@@ -0,0 +1,95 @@
+        uses: ./.github/actions/create-package-validation-issue
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          validation-report-json: 'validation-report.json'
+          validation-report-txt: 'validation-report.txt'
+          workflow-run-number: ${{ github.run_number }}
+          workflow-run-id: ${{ github.run_id }}
+          server-url: ${{ github.server_url }}
+          repository: ${{ github.repository }}
+            
+      - name: Post Issue Summary
+        if: steps.create_issue.conclusion == 'success' && steps.create_issue.outputs.issue-url != ''
```

actionlint (1.7.7)

property "issue-url" is not defined in object type {}

(expression)